src/components/Footer.js

```diff
@@ -0,0 +1,33 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import { translate } from 'react-i18next';
+import { Link } from 'gatsby-plugin-i18next';
+
+class Footer extends Component {
+  static propTypes = {
+    siteTitle: PropTypes.string.isRequired,
+    t: PropTypes.string
```

Comment:
`t` is used like a function in the render method but its prop type is string here. Also, because it's not `.isRequired` you don't have to provide the default prop. An empty string can't be invoked so it'd cause an error if it were ever actually called on.